### PR TITLE
[MOBILE-509] Fix dev environment

### PR DESCRIPTION
**Pull request by rlepinski**
### What do these changes do?
Make it possible to develop the plugin without copying files over. We are using the approach outlined in https://medium.com/@charpeni/setting-up-an-example-app-for-your-react-native-library-d940c5cf31e4

### Why are these changes necessary?
Makes development easier

### How did you verify these changes?
Ran app and modified code.

### CODE REVIEW

File: sample/AirshipSample/android/build.gradle [hunk 1]
```diff
@@ -22,18 +23,12 @@ buildscript {
 
 allprojects {
     repositories {
-        google()
```
Comment (crow) [inline sample/AirshipSample/android/build.gradle]: this ordering is important, was this on purpose?

File: sample/AirshipSample/android/gradlew.bat [hunk 2]
```diff
@@ -1,3 +1,19 @@
+@rem
+@rem Copyright 2015 the original author or authors.
```
Comment (crow) [inline sample/AirshipSample/android/gradlew.bat]: do we need the correct year for these apache copyrights headers?

Comment (rlepinski) [reply sample/AirshipSample/android/gradlew.bat]: this is generated, so no

File: sample/AirshipSample/android/build.gradle (hunk 1 above)
Comment (rlepinski) [reply sample/AirshipSample/android/build.gradle]: mavenLocal() normally is first, so its fine. Just needs to be ahead of jcenter()

File: sample/AirshipSample/package.json [hunk 3]
```diff
@@ -1,18 +1,25 @@
 {
   "name": "AirshipSample",
-  "version": "3.0.0",
+  "version": "0.0.1",
```
[the remark this answers is not in the document]
Comment (rlepinski) [reply sample/AirshipSample/package.json]: The sample does not need to be versioned. Probably updated when I regenerated the template

File: sample/AirshipSample/__tests__/App-test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/**
+ * @format
+ */
+
+import 'react-native';
+import React from 'react';
+import App from '../App';
+
+// Note: test renderer must be required after react-native.
```
[the remark this answers is not in the document]
Comment (rlepinski) [reply sample/AirshipSample/__tests__/App-test.js]: Another generated file

**Pull request merged**